doc/protocol.md

```diff
@@ -1072,6 +1072,13 @@ The following options can be specified in the "open" control message:
    you don't set this field, the actual tag will not be checked.  To
    express that you expect the file to not exist, use "-" as the tag.
 
+* "attrs": a JSON object containing file owner and group information
+           to set. Supporting the file ownership fields returned by `fsinfo`.
```

Comment:
Inconsistent indentation.

"Supporting" sounds strange. "This supports the file ownership fields..."?